### ISSUE TITLE
Not working on 1.1.72

### ISSUE THREAD

**Issue by nenoro** [abba23/spotify-adblock#53]
Hello

So i'm on gentoo i have the 1.1.72 installed and LD_PRELOAD doesn't work i have tried with LD_PRELOAD_PATH it launches but it doesn't remove the banner ad and cut the sound when a ad is playing.

also i have this error when i try 

- LD_PRELOAD=/usr/local/lib/spotify-adblock.so /usr/bin/spotify
- LD_PRELOAD=/usr/local/lib/spotify-adblock.so spotify

/bin/sh: symbol lookup error: /usr/local/lib/spotify-adblock.so: undefined symbol: cef_string_userfree_utf16_free

It worked so well before.

Don't know what to do now

**Comment by abba23** [comment]
See #46. You need to specify the actual Spotify binary, not some wrapper script.

**Comment by nenoro** [comment]
trying to understand your answer the binary is 1.1.72.439.gc253025e if is that you ask

**Comment by abba23** [comment]
I explained it in the issue I linked. Your `/usr/bin/spotify` file is a shell script, not the Spotify binary.

You'll need to figure out the path to the actual Spotify binary (e.g. by looking at the script using `cat /usr/bin/spotify`) and use that path instead of `/usr/bin/spotify` when trying to launch Spotify with the adblocking library preloaded.

Depending on what else the script does, you might also need to do some other things like setting `LD_LIBRARY_PATH`.
